vdev/src/git.rs

```diff
@@ -51,6 +51,11 @@ pub fn list_files() -> Result<Vec<String>> {
         .collect())
 }
 
+pub fn get_git_sha() -> Result<String> {
+    let output = capture_output(&["rev-parse", "--short", "HEAD"])?;
+    Ok(output.trim_end().to_string())
```

Comment:
If you want to map it, this could be
```suggestion
    capture_output(&["rev-parse", "--short", "HEAD"])
        .map(|output| output.trim_end().to_string())
```